floater/test/test_utils.py

```diff
@@ -95,24 +95,37 @@ def test_floats_to_netcdf(tmpdir, mitgcm_float_datadir_csv):
 
     input_dir = str(mitgcm_float_datadir_csv) + '/'
     output_dir = str(tmpdir) + '/'
-
     os.chdir(input_dir)
+
+    # least options
+    utils.floats_to_netcdf(input_dir=input_dir, output_fname='test')
+    # most options
     utils.floats_to_netcdf(input_dir=input_dir, output_fname='test',
                            float_file_prefix='float_trajectories',
-                           ref_time=None, step_time=86400,
+                           ref_time='1993-01-01', step_time='86400',
                            output_dir=output_dir, output_prefix='prefix_test')
 
     # filename prefix test
+    os.chdir(input_dir)
+    mfdl = xr.open_mfdataset('test.nc/float_trajectories.*.nc')
     os.chdir(output_dir)
-    mfd = xr.open_mfdataset('test.nc/prefix_test.*.nc')
+    mfdm = xr.open_mfdataset('test.nc/prefix_test.*.nc')
```

Comment:
@rabernat Yes, I include `_maybe_add_suffix(output_fname, '.nc')` in `floats_to_netcdf` to label the `--output_format` in case users using different formats for the same MITgcm output. What about change it to `_maybe_add_suffix(output_fname, '.netcdf')`? `output_dir` is the root directory of the output folder (by default `./`), and `output_fname` (a key option, no default value) is the name of the folder to be created. Do you think it is redundant to keep both?